lib/ddtrace/span.rb

```diff
@@ -269,36 +271,36 @@ def to_msgpack(packer = nil)
       if !@start_time.nil? && !@end_time.nil?
         packer.write_map_header(13) # Set header with how many elements in the map
 
-        packer.write(:start)
+        packer.write('start')
```

Comment:
worth adding a comment on why we use strings instead of symbols?

any benefits to making these constants/freezing them outside the scope of this method?

Reply:
> any benefits to making these constants/freezing them outside the scope of this method?

Instead of doing that, given there are so many strings,  I added `# frozen_string_literal: true` to the top of the file, which freezes all strings in this. I did check all strings declared in this file and they are all safe to freeze.

I also benchmark only the `# frozen_string_literal: true` change, to see if the other strings frozen in the file would make a change to our numbers, but they didn't, so the performance improvement does come from the change from symbol to string only.

> worth adding a comment on why we use strings instead of symbols?

I'll add a comment for that, good call!